bin/fluid

```diff
@@ -108,6 +118,42 @@ class FluidCommandLine {
 	}
 
 	/**
+	 * @param \TYPO3Fluid\Fluid\Core\Cache\FluidCacheWarmupResult $result
+	 * @return string
+	 */
+	protected function renderWarmupResult(\TYPO3Fluid\Fluid\Core\Cache\FluidCacheWarmupResult $result) {
+		$string = PHP_EOL . 'Template cache warmup results' . PHP_EOL . PHP_EOL;
+		foreach ($result->getResults() as $templatePathAndFilename => $aspects) {
+			$string .= sprintf(
+				"%s\n    Compiled? %s\n    Has Layout? %s\n",
+				$templatePathAndFilename,
+				$aspects[\TYPO3Fluid\Fluid\Core\Cache\FluidCacheWarmupResult::RESULT_COMPILABLE] ? 'YES' : 'NO',
+				$aspects[\TYPO3Fluid\Fluid\Core\Cache\FluidCacheWarmupResult::RESULT_HASLAYOUT] ? 'YES' : 'NO'
+			);
+			if ($aspects[\TYPO3Fluid\Fluid\Core\Cache\FluidCacheWarmupResult::RESULT_COMPILABLE]) {
+				$string .= sprintf(
+					"    Compiled as: %s\n",
+					$aspects[\TYPO3Fluid\Fluid\Core\Cache\FluidCacheWarmupResult::RESULT_COMPILEDCLASS]
```

Comment:
It is correct; the compiled class name is only output if the template is compilable, but it is set (to the template identifier) also if the template is not compilable.